server/src/main/java/org/opensearch/search/backpressure/SearchBackpressureService.java

```diff
@@ -150,7 +149,6 @@ public SearchBackpressureService(
         this.taskResourceTrackingService = taskResourceTrackingService;
         this.taskResourceTrackingService.addTaskCompletionListener(this);
         this.threadPool = threadPool;
-        this.timeNanosSupplier = timeNanosSupplier;
```

Comment:
Could we please remove `timeNanosSupplier` from constructor args?

Reply:
It is used below in line 158.

Reply:
Sorry missed that, thank you